internal/local/command_kafka_start.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
+var defaultKafkaRestPorts = []string{"8082", "9092", "10012", "20012"}
```

Comment:
9092 is typically used for the broker, not Kafka REST: https://docs.confluent.io/platform/current/installation/system-requirements.html#ports